src/main/java/jenkins/plugins/logstash/LogstashBuildWrapper.java

```diff
@@ -29,14 +29,21 @@
 import hudson.model.BuildListener;
 import hudson.model.AbstractBuild;
 import hudson.model.AbstractProject;
+import hudson.model.BuildableItemWithBuildWrappers;
 import hudson.tasks.BuildWrapper;
 import hudson.tasks.BuildWrapperDescriptor;
 
 import java.io.IOException;
 import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.List;
 
 import org.kohsuke.stapler.DataBoundConstructor;
 
+import com.michelin.cio.hudson.plugins.maskpasswords.MaskPasswordsBuildWrapper;
+import com.michelin.cio.hudson.plugins.maskpasswords.MaskPasswordsBuildWrapper.VarPasswordPair;
+import com.michelin.cio.hudson.plugins.maskpasswords.MaskPasswordsConfig;
+
```

Comment:
We have encountered issues with various plugins when using the mask password plugin. Could we use reflection to perform mask-password logic only if the mask password plugin is installed?